### ISSUE TITLE
Upload of wk.Annotation should require organization_id

### ISSUE THREAD

**Issue by fm3** [scalableminds/webknossos-libs#1015]
When creating a new annotation via `a = wk.Annotation(name, dataset_name=dataset_name, voxel_size=voxel_size)` it should be required to pass an organization name for the specified dataset to avoid ambiguity.

Otherwise during upload to wk, the lookup can be ambiguous and may fail if other organizations have datasets of that name.

Related: https://github.com/scalableminds/webknossos/issues/7681

**Comment by hotzenklotz** [comment]
I think it still would be better to infer that somehow. There are not many users which are part of multiple organisations. This would make the API more difficult for them.

The other question is: Are organizations referenced by name or ID?

**Comment by fm3** [comment]
> I think it still would be better to infer that somehow

True, if there is an active webknossos context, the orga of the user can be used automatically.

> Are organizations referenced by name or ID

This is the biggest mess… They are referenced by name (not the display name) (the name that is also the orga directory name in the binaryData directory). However, that field is called organization_id in the libs. I once tried to adapt the webknossos side to do the same, and drop the mongoId entirely in https://github.com/scalableminds/webknossos/pull/7386 but @normanrz stopped me because some external references still use the mongoId values.

**Comment by markbader** [comment]
As discussed in PR #1155 the organization name is not necessary for construction but should be inferred or requested from the user when uploading the Annotation.
